components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1943,6 +1943,7 @@ void JsonRpcService::GetSupportsInterface(
     std::move(callback).Run(
         false, mojom::ProviderError::kInvalidParams,
         l10n_util::GetStringUTF8(IDS_WALLET_INVALID_PARAMETERS));
+    return;
```

Comment:
Same as above concerns - this is changing the behavior. This is fixing it IMO, but will result in callback not happening below anymore

Reply:
Explained above.

Reply:
Marking as resolved; thanks for the replies on above 😄

Reply:
This is a correct fix.